packages/css/src/plugins/sort-atomic-style-sheet.ts

```diff
@@ -1,20 +1,40 @@
-import type { ChildNode, Node, Rule, Plugin } from 'postcss';
+import type { ChildNode, Rule, Plugin, AtRule } from 'postcss';
 
-import { styleOrder } from '../utils/style-ordering';
+import { sortPseudoSelectors } from '../utils/sort-pseudo-selectors';
 
-/**
- * Returns where it should be placed.
- *
- * @param selector
- * @returns
- */
-const pseudoSelectorScore = (selector: string) => {
-  const index = styleOrder.findIndex((pseudoClass) => selector.trim().endsWith(pseudoClass));
-  return index + 1;
+import { parseAtRule } from './at-rules/parse-at-rule';
+import { sortAtRules } from './at-rules/sort-at-rules';
+import type { AtRuleInfo } from './at-rules/types';
+
+const sortAtRulePseudoSelectors = (atRule: AtRule) => {
```

Comment:
Brought this in to reduce duplicate behaviour?

Reply:
ye